### PR TITLE
fix default check of "don't show anymore" + move wizard to App.tsx

### LINKED ISSUE #2253: wizard keep reappearing when switching tabs
the wizard should reappear at next app launch, not constantly during usage of the current application launch

Screenshot 2024-05-20 at 10 53 09

Screenshot 2024-05-20 at 10 53 23

### LINKED ISSUE #2271: Wizard Don't show anymore should be opt in, not opt out
By default the Don't show anymore bow is thicker. It should be not selected by default.

In other words, default should be that the Wizard appears until the user perform the action to check the box for it to do not show anymore.

### PR DESCRIPTION
**Pull request by arthur-lemeur**
Fixes #2271  Fixes #2253

### CODE REVIEW

File: src/renderer/library/components/Wizard.tsx [hunk 1]
```diff
@@ -100,8 +103,27 @@ export const WizardModal = () => {
...
                         </Tabs.Trigger>
                         <div style={{display: "flex", alignItems: "center", gap: "10px", position: "absolute", bottom: "30px", left: "30px"}}>
-                            <input type="checkbox" checked={checked} onChange={() => setChecked(!checked)} id="wizardCheckbox" name="wizardCheckbox" />
-                            <label htmlFor="wizardCheckbox">{__("wizard.dontShow")}</label>
+                            <input type="checkbox" checked={checked} onChange={() => setChecked(!checked)} id="wizardCheckbox" name="wizardCheckbox" className={stylesGlobal.checkbox_custom_input} />
+                            <label htmlFor="wizardCheckbox" className={stylesGlobal.checkbox_custom_label}>
+                                <div
+                                    tabIndex={0}
+                                    role="checkbox"
+                                    onKeyDown={(e) => {
```
Comment (danielweck) [inline src/renderer/library/components/Wizard.tsx:111]: `onKeyUp` please :)
(`onKeyDown` is repeatable which can have adverse UX effects)

File: src/renderer/library/components/Wizard.tsx [hunk 2]
```diff
@@ -100,8 +103,27 @@ export const WizardModal = () => {
...
+                                        if (e.key === " ") {
```
Comment (danielweck) [inline src/renderer/library/components/Wizard.tsx:112]: "Space" please (not " ") for consistency with existing code

**Review by danielweck — CHANGES_REQUESTED** [review]

File: src/renderer/library/components/Wizard.tsx [hunk 3]
```diff
@@ -61,7 +64,7 @@ export const WizardModal = () => {
     const dispatch = useDispatch();
     const opened = useSelector((state: ILibraryRootState) => state.wizard.opened);
 
-    const [checked, setChecked] = React.useState(true);
+    const [checked, setChecked] = React.useState(false);
```
Comment (danielweck) [inline src/renderer/library/components/Wizard.tsx:67]: I do not know the interaction between `state.wizard.opened` and the local `checked` state so I will let @panaC review this part.

Comment (panaC) [reply src/renderer/library/components/Wizard.tsx:67]: `state.wizard.opened` is the persisted opened state
`checked` is the local checkbox state mounted once

Comment (danielweck) [reply src/renderer/library/components/Wizard.tsx:67]: sure, but how/when is `state.wizard.opened` mutated?
reading the code with my untrained eyes, I only see a read, not a write

File: src/renderer/library/components/Wizard.tsx (hunk 2 above)
Comment (panaC) [reply src/renderer/library/components/Wizard.tsx]: @danielweck I see something strange on your PR the "Space" is on the `key` value but on my test that works only with the `code` value not `key` ("Space" not referenced in https://www.w3.org/TR/uievents-key/#named-key-attribute-values ) 
And if I follow your PR (https://github.com/edrlab/thorium-reader/pull/2269/files) you replace `" "` on `key` value. 

Is it working on keyPress and not on keyDown/keyUp ?

Comment (panaC) [reply src/renderer/library/components/Wizard.tsx]: see control character : https://www.w3.org/TR/uievents-key/#control

File: src/renderer/library/components/Wizard.tsx (hunk 3 above)
Comment (panaC) [reply src/renderer/library/components/Wizard.tsx:67]: https://github.com/edrlab/thorium-reader/blob/2a9f7b2a962cb8ef80a2dcbc9d453144bdb8dec3/src/renderer/library/components/Wizard.tsx#L71

**Comment by panaC** [comment]
I merge this PR it's work for me

**Pull request merged**